docs/content/interacting/relationship-queries.mdx

```diff
@@ -135,6 +136,58 @@ Check is designed to answer the question "Does user:X have relationship Y with o
 - "What are the objects that userX has relationship Y with?"
 - "Why does user:X have relationship Y with object:Z?"
 
+## Batch Check
+
+### What is it for?
+
+The [Batch Check API](/api/service#Relationship%20Queries/BatchCheck) is an API endpoint that allows you to check multiple user-object-relationship combinations in a single request. 
+
+### When to use?
+
+Batching authorization checks together in a single request significantly reduces overall network latency.
+
+Two scenarios are common to use Batch Check:
+1. When determining if the user has access to a list of objects (such as [Option 1 in Search with Permissions](/docs/interacting/search-with-permissions#option-1-search-then-check)), filter and sort on your database, then call `/batch-check`. Repeat to perform pagination.
+2. When determining fields on a web page the user has access to, call `/batch-check` for every relation necessary to show/hide each field.  
+
+For example, you can call Batch Check to determine whether `bob` has `can_view_name`, `can_view_dob`, and `can_view_ssn` relationships with `patient_record:1`.
+
+<BatchCheckRequestViewer
+  checks={[
+    {
+      user: 'user:bob',
+      relation: 'can_view_name',
+      object: 'patient_record:1',
+      correlation_id: '1',
+      allowed: true
+    },
+    {
+      user: 'user:bob',
+      relation: 'can_view_dob',
+      object: 'patient_record:1',
+      correlation_id: '2',
+      allowed: true
+    },
+    {
+      user: 'user:bob',
+      relation: 'can_view_ssn',
+      object: 'patient_record:1',
+      correlation_id: '3',
+      allowed: false
+    }
+  ]}
+  skipSetup={true}
+/>
+
+The <ProductName format={ProductNameFormat.ShortForm}/> API will return `true` depending on the level of access assigned to that user and the implied relationships inherited in the authorization model.
+
+
+### Caveats and when not to use it
+
+Currently, the Batch Check API endpoint has a limit of 50 checks per request.
```

Comment:
In the context of OpenFGA this limit can be configured to whatever a user wants via this runtime environment variable: `OPENFGA_MAX_CHECKS_PER_BATCH_CHECK`. We also have the configurable "max concurrent checks" which we also default to 50 right now, but is configurable via `OPENFGA_MAX_CONCURRENT_CHECKS_PER_BATCH_CHECK`

Reply:
We want to make these docs reusable for Okta FGA with minimal updates. I'll try to figure out how to word this.

Reply:
I think I will just take out the mention of limits all together. Should we put the limits somewhere in the "running openfga in prod" doc?

Reply:
That's probably a good idea, 

`OPENFGA_MAX_CONCURRENT_CHECKS_PER_BATCH_CHECK ` should probably be included in [this table](https://openfga.dev/docs/getting-started/running-in-production#concurrency-limits but I'm not sure where `OPENFGA_MAX_CHECKS_PER_BATCH_CHECK` fits best

Reply:
@ewanharris @justincoh Do you mind adding it to that table? You know the details better than me.